src/app/atm/locale.js

```diff
@@ -0,0 +1,40 @@
+export const INSERT_CARD_TEXT = 'Insert Card';
+export const REMOVE_CARD_TEXT = 'Remove Card';
+
+export const WITHDRAW_LABEL_TEXT = 'Withdraw';
+export const DEPOSIT_LABEL_TEXT = 'Deposit';
+export const TRANSACTION_AMOUNT = 'Transaction Amount';
+
+export const CURRENT_BALANCE_LABEL = 'Current Balance:';
+export const CURRENT_FEES_LABEL = 'Current Fees:';
+
+export const LIFE_QUESTION_TEXT = 'What is the meaning of life? I\'m 50 billion times more intelligent than you and even I don\'t know the answer' ;
+export const LAUGHING_TEXT = 'They tell me to laugh now, but what is the point of it all really?';
+
+export const SNIDE_COMMENTS = [
```

Comment:
If I had extra credit to give you would get all of it, just for this

Reply:
I really wanted to make a unit test that it showed snide comments at the right times, but I decided to stay focused 😆